Add meta descriptions to Lisk-SDK, closes #371 (#387)
LiskHQ/lisk-docs: modules/ROOT/pages/lisk-elements/packages/api-client/transactions.adoc

= Lisk Elements API Client: Transactions Resource
Mona Bärenfänger <mona@lightcurve.io>

> :description: Technical reference for the Transactions endpoints of the API Client package of Lisk Elements. It copmrises Usage examples, available parameters and example responses.

Reviewer: ```suggestion
:description: Technical references regarding the Transactions endpoints of the API Client package of Lisk Elements.This consists of usage examples, available parameters and example responses.
```

> :description: Technical references regarding the Transactions endpoints of the API Client package of Lisk Elements.This consists of usage examples, available parameters and example responses.

Reviewer: ```suggestion
:description: Technical references regarding the Transactions endpoints of the API Client package of Lisk Elements. This consists of usage examples, available parameters and example responses.
```
